compiler/modules/nback/function.bal

```diff
@@ -41,83 +41,113 @@ final RuntimeFunction addToRestArgsFunction = {
     attrs: []
 };
 
-function buildCall(llvm:Builder builder, Scaffold scaffold, bir:CallInsn insn) returns BuildError? {
-    var { symbol: funcSymbol, erasedSignature, signature } = insn.func;
+type FunctionConstDefn record {|
```

Comment:
@jclark, decided to go with `FunctionConstDefn` (and `FunctionPointerDefn`), to avoid confusion with already existing [`buildFunctionConstValue`](https://github.com/ballerina-platform/nballerina/blob/45f26a5b7aa8dca5e811c88573665da72f0859b1/compiler/modules/nback/build.bal#L350)

Reply:
Ignore above